Source/Core/AbsyType.cs

```diff
@@ -3160,6 +3160,18 @@ public CtorType(IToken/*!*/ token, TypeCtorDecl/*!*/ decl, List<Type>/*!*/ argum
       return QKeyValue.FindBoolAttribute(Decl.Attributes, "datatype");
     }
 
+    // This attribute is used to provide a user-specified SMT-LIB declaration for the type.
+    public string getBuiltinDecl() {
+      return this.Decl.FindStringAttribute("builtindecl");
```

Comment:
Boogie code follows the naming convention of starting method names with Capital letters.

Reply:
Fixed